bin/crew

```diff
@@ -124,6 +124,25 @@ String.use_color = args['--color'] || !args['--no-color']
 @verbose           = @opt_verbose ? 'v' : ''
 @short_verbose     = @opt_verbose ? '-v' : ''
 
+class ExitMessage
+  attr_accessor :msg
```

Comment:
Making the `msg` variable available outside the class should not be necessary (this line is not needed)